src/store.js

```diff
@@ -36,6 +37,10 @@ const mutations = {
 
   deleteKey(state, key) {
     state.data = state.data.filter(item => item.key !== key);
+  },
+
+  setRecentlyOpenedPaths(state, paths) {
+    state.recentlyOpenedPaths = [...paths];
```

Comment:
```suggestion
    state.recentlyOpenedPaths = paths;
```

No need to spread and create a new array.

Reply:
updated